src/main/resources/api.yaml

```diff
@@ -1177,6 +1177,14 @@ paths:
         409:
           description: "The user already exists."
 
+  /testing-support/deleteAll:
```

Comment:
this should probably be a delete operation on test-data
so /testing-support/test-data
operation delete
POST on a endpoint with a method name is more RPC than REST

Reply:
applied the suggestion and pushed. Thanks